src/main/webapp/academicAdminOffice/chooseStudentEnrolmentExecutionPeriod.jsp

```diff
@@ -23,6 +23,8 @@
 <%@ taglib uri="http://struts.apache.org/tags-logic" prefix="logic"%>
 <%@ taglib uri="http://fenix-ashes.ist.utl.pt/fenix-renderers" prefix="fr" %>
 <%@ taglib uri="http://fenix-ashes.ist.utl.pt/taglib/academic" prefix="academic" %>
+<%@ page isELIgnored="true"%>
```

Comment:
You could avoid this by just escaping the new `${}` entries you are creating :smile: